radio/src/targets/taranis/bootloader/boot_menu.cpp

```diff
@@ -39,7 +39,11 @@ void bootloaderInitScreen()
   lcdSetContrast(true);
 
   backlightInit();
+#if defined(LCD_BRIGHTNESS_DEFAULT)
+  backlightEnable(LCD_BRIGHTNESS_DEFAULT);
```

Comment:
Is this good for the MT12 also? as it also defines LCD_BRIGHTNESS_DEFAULT

Reply:
yes, ok on mt12 too (or any radio that defines it really)

Reply:
Which is still leaves MT12... it is the one and only :rofl: